wiki/Getting-Started-UIKit.md

```diff
@@ -28,7 +28,16 @@ class FirstViewController: UIWorkflowItem<String, String>, FlowRepresentable { /
     required init(with name: String) { // SwiftCurrent
         self.name = name
         super.init(nibName: nil, bundle: nil)
+        configureViews()
+    }
+
+    required init?(coder: NSCoder) { nil }
 
+    @objc private func savePressed() {
+        proceedInWorkflow(emailTextField.text ?? "") // SwiftCurrent
+    }
+
+    private func configureViews() {
```

Comment:
From a consumability standpoint I like this better because it's easier to see the sections that are required for SwiftCurrent and what is set up code for the example.